File: output/ChatGPT/avatar/Python/C++/atcoder_ABC125_B.cpp
#include <iostream>
#include <vector>

int main() {
    int N;
    std::cin >> N;
    
    std::vector<int> V(N);
    for (int i = 0; i < N; i++) {
        std::cin >> V[i];
    }
    
    std::vector<int> C(N);
    for (int i = 0; i < N; i++) {
        std::cin >> C[i];
    }
    
    std::vector<int> list(1);
    int ans = 0;
    int X = 0;
    int Y = 0;
    for (int i = 0; i < N; i++) {
        if (V[i] - C[i] > 0) {
            X = V[i];
            Y = C[i];
            list.push_back(X - Y);
        }
    }
    
    int sum = 0;
    for (int i = 0; i < list.size(); i++) {
        sum += list[i];
    }
    
    std::cout << sum << std::endl;

    return 0;
}

File: output/ChatGPT/avatar/Python/Java/atcoder_ABC125_B.java
import java.util.Scanner;

public class atcoder_ABC125_B {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int N = Integer.parseInt(scanner.nextLine());
        String[] V = scanner.nextLine().split(" ");
        int[] VInt = new int[N];
        for (int i = 0; i < N; i++) {
            VInt[i] = Integer.parseInt(V[i]);
        }

        String[] C = scanner.nextLine().split(" ");
        int[] CInt = new int[N];
        for (int i = 0; i < N; i++) {
            CInt[i] = Integer.parseInt(C[i]);
        }

        int[] list = new int[N + 1];
        int ans = 0;
        int X = 0;
        int Y = 0;
        for (int i = 0; i < N; i++) {
            if (VInt[i] - CInt[i] > 0) {
                X = VInt[i];
                Y = CInt[i];
                list[i + 1] = X - Y;
            }
        }

        int sum = 0;
        for (int i = 0; i < list.length; i++) {
            sum += list[i];
        }

        System.out.println(sum);
    }
}